src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -23,219 +23,6 @@ protected override SyntaxTree ParseTree(string text, CSharpParseOptions options)
             return SyntaxFactory.ParseSyntaxTree(text, options ?? TestOptions.Regular);
         }
 
-        [Fact]
-        public void RecordParsing01()
```

Comment:
>RecordParsing01 [](start = 20, length = 15)

Why delete these tests?

Reply:
Woops, did not mean to, just meant to consolidate into a single file. Re-added.